File: test/ayush.dev/scala/pipelines/CreatedFromUsedAccount/code/src/main/scala/io/prophecy/pipelines/createdfromusedaccount/graph/src_avro.scala
package io.prophecy.pipelines.createdfromusedaccount.graph

import io.prophecy.libs._
import io.prophecy.pipelines.createdfromusedaccount.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object src_avro {

  def apply(spark: SparkSession): DataFrame = {
    import org.apache.avro.Schema
    var reader = spark.read.format("avro")
    reader = reader
    reader.load("dbfs:/Prophecy/qa_data/avro/CustomersDatasetInput.avro")
  }

}

File: test/ayush.dev/python/pipelines/TestNew1/code/testnew1/graph/src_avro.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from testnew1.config.ConfigStore import *
from testnew1.udfs.UDFs import *

def src_avro(spark: SparkSession) -> DataFrame:
    return spark.read.format("avro").load("dbfs:/Prophecy/qa_data/avro/CustomersDatasetInput.avro")
